PaymentSystem.Server.UnitTests/CreateWalletCommandHandlerTests.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using IdentityServer4.EntityFramework.Options;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using NUnit.Framework;
+using PaymentSystem.Server.Application.Currencies;
+using PaymentSystem.Server.Application.Promotion;
+using PaymentSystem.Server.Application.Wallets.Commands;
+using PaymentSystem.Server.Data;
+using PaymentSystem.Server.Models;
+
+namespace PaymentSystem.Server.UnitTests
+{
+    public class CreateWalletCommandHandlerTests
+    {
+        private ApplicationDbContext context;
```

Comment:
Isn't `ApplicationDbContext` implementing `IDisposable`? If so, it must be called.
Just like there's a SetUp, there's TearDown where you can call `context.Dispose()`

Reply:
It is indeed! Thanks for the heads-up! Fixed it in the newer commit